packages/react/etc/components-react.api.md

```diff
@@ -1175,6 +1185,24 @@ export interface VideoTrackProps extends React_2.VideoHTMLAttributes<HTMLVideoEl
     trackRef?: TrackReference;
 }
 
+// @alpha (undocumented)
+export interface VoiceAssistant {
+    // (undocumented)
+    agent: RemoteParticipant | undefined;
+    // (undocumented)
+    agentTranscriptions: ReceivedTranscriptionSegment[];

```

Comment:
this would be the agent's text, right? the transcribed user audio would still be to be gotten out of a different place?

Reply:
yeah, the user's transcriptions would still be tied to that user, via `useTrackTranscriptions(localParticipantAudioTrack)`

Reply:
do you think it would make more sense to include local participant transcriptions + tracks here as well?

Reply:
not sure - would want to see how it works in practice. i think threading the agent and user transcriptions against eachother in 1:1 setting (and even potentially in non-1:1 settings) will be a common case but i don't think the way it's set up is a problem

Reply:
I like this idea.. I wonder if they should be combined at the protocol level? (i.e. renaming Transcriptions to be generic TextSegment? and sending chat through it as well?)

Reply:
That's an interesting idea! 
I like that this would mean a protocol-level definition for chat messages (which we currently do not have). 
But I think it would still be good to keep transcriptions and chat messages distinguishable. 
Especially when thinking about a first class chat in the future.

Reply:
+1 to this concept - with some sort of type/kind/source flag that indicates where the text came from (direct input, transcribed from audio, read from OCR, etc)

And for what it's worth when building my own app on this sort of thing it was really important to distinguish between spoken-then-transcribed text and typed-text when using it in LLM inputs because you need to consider transcriptions to be of lower accuracy than typed inputs (e.g. if I typed "Lukas" in one spot and also spoke the name which got transcribed as "Lucas" somewhere else, the app needs to be able to know that the correct spelling is "Lukas").